### PR TITLE
LandingPage ContentType modification caution

### PR DESCRIPTION
**Pull request by kmadejski**
Information about possible errors due to the Landing Page ContentType modification should be better visible to the reader.

### CODE REVIEW

File: docs/guide/field_type_reference.md [hunk 1]
```diff
@@ -1214,7 +1214,9 @@ $keywordValue = new Value( "php5,css3,html5" );
 
     Landing Page Field Type is only used in the Landing Page Content Type that is included in eZ Enterprise.
 
-    The structure of the Landing Page Content Type should not be modified, as it may cause errors.
+    !!! caution
+    
+        The structure of the Landing Page Content Type should not be modified, as it may cause errors.
```
Comment (arfaram) [inline docs/guide/field_type_reference.md]: It is more on pre-defined FieldTypes within the LandingPage contentType. Of course you are able to add new FieldTypes to the LandingPage but don't change any existing identifier (name, description,page)

**Comment by kmadejski** [comment]
Thanks, @arfaram! I've improved the message, please check whether it's fine now.

File: docs/guide/field_type_reference.md [hunk 2]
```diff
@@ -1214,7 +1214,9 @@ $keywordValue = new Value( "php5,css3,html5" );
 
     Landing Page Field Type is only used in the Landing Page Content Type that is included in eZ Enterprise.
 
-    The structure of the Landing Page Content Type should not be modified, as it may cause errors.
+    !!! caution
+    
+        The default Field Type identifiers of the Landing Page Content Type should not be modified, as it may cause fatal errors. The new Field Types can be added safely but shouldn't be required.
```
Comment (DominikaK) [inline docs/guide/field_type_reference.md]: Shouldn't it be "New Fields can be added"? (Or maybe "Field of other types"?)

**Comment by kmadejski** [comment]
@DominikaK the caution message has been changed as we discussed.

**Comment by arfaram** [comment]
@kmadejski it looks much better

**Pull request merged**